clients/python/src/model_registry/_client.py

```diff
@@ -17,27 +19,55 @@ class ModelRegistry:
     def __init__(
         self,
         server_address: str,
-        port: int,
+        port: int = 443,
+        *,
         author: str,
-        client_key: str | None = None,
-        server_cert: str | None = None,
-        custom_ca: str | None = None,
+        user_token: bytes | None = None,
```

Comment:
Not a requirement right now, but we should open another issue to add support for client certificate as well for mutual authentication.